.browserslistrc

```diff
@@ -0,0 +1,3 @@
+# Browsers that we support
+
+last 4 version
```

Comment:
Is this also implying the last 4 versions of IE? I'm not exactly sure about the browserlint syntax in context of vendor prefixes.

Reply:
Ahhhh yes. Who the hell is using IE8?

I've been reading up a little on [browserlist](https://github.com/browserslist/browserslist). If we put `defaults` instead of `last 4 version` then the following settings are used:

* `> 0.5%` - versions with a global marketshare greater than 0.5%
* `last 2 versions` - the last 2 versions of a browser
* `Firefox ESR` -  the latest [Firefox ESR] version
* `not dead` - versions that aren't dead e.g. IE10 is considered dead

Do you think this would be sufficient? You can see the browsers that will be supported by this by using [browserl.ist](https://browserl.ist/?q=defaults).

It might be worth having a discussion on what browsers PaperCSS should support before moving forward with this PR.